### PR TITLE
Fix cluster profiles for CBO and release jobs

### PR DESCRIPTION
**Pull request by derekhiggins**
Switch over assistet and sno jobs to the correct profile

### CODE REVIEW

File: ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml [hunk 1]
```diff
@@ -26089,7 +26089,7 @@ periodics:
     repo: release
   labels:
     ci-operator.openshift.io/cloud: packet
-    ci-operator.openshift.io/cloud-cluster-profile: packet-assisted
+    ci-operator.openshift.io/cloud-cluster-profile: packet
```
Comment (andfasano) [inline ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml:26092]: @derekhiggins is this ok (not very clear from this diff)?

Comment (derekhiggins) [reply ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml:26092]: Yes, I switched one job(nightly-4.6, e2e-metal-ipi ) back to packet in 
 job in ci-operator/config/openshift/release/openshift-release-master__nightly-4.6.yaml

Comment (andfasano) [reply ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml]: Thanks

**Comment by andfasano** [comment]
@lranjbar could you please check if the assisted jobs failure are expected (I'd say so, given that just the cluster profile changed)

**Review by lranjbar — COMMENTED** [review]
The error is unrelated to the profile change but if it isn't too much trouble to update the base images while we are here:

```
base_images:
  assisted-image-service:
    name: ocm-2.5
    namespace: edge-infrastructure
    tag: assisted-image-service
  assisted-installer:
    name: ocm-2.5
    namespace: edge-infrastructure
    tag: assisted-installer
  assisted-installer-agent:
    name: ocm-2.5
    namespace: edge-infrastructure
    tag: assisted-installer-agent
  assisted-installer-controller:
    name: ocm-2.5
    namespace: edge-infrastructure
    tag: assisted-installer-controller
  assisted-service:
    name: ocm-2.5
    namespace: edge-infrastructure
    tag: assisted-service
  assisted-test-infra:
    name: ocm-2.5
    namespace: edge-infrastructure
    tag: assisted-test-infra
```

**Comment by lranjbar** [comment]
Otherwise :+1: since errors are not related to this change

**Comment by andfasano** [comment]
@wking could you please have a look at these changes? Thanks

**Comment by andfasano** [comment]
I think we can close it, addressed by #25355

**Pull request closed without merging**